third_party/terraform/tests/resource_pubsub_subscription_test.go

```diff
@@ -219,8 +219,9 @@ resource "google_pubsub_topic" "foo" {
 }
 
 resource "google_pubsub_subscription" "foo" {
-  name  = "%s"
-  topic = google_pubsub_topic.foo.id
+  name   = "%s"
+  topic  = google_pubsub_topic.foo.id
+  filter = "attributes.foo = "bar""
```

Comment:
The quotes here will need to be escaped, probably something like this:
```
filter = "attributes.foo = \"bar\""
```

Otherwise terraform will complain that this config is invalid